src/api.cc

```diff
@@ -1196,3 +1219,41 @@ static void checkBundleTag(const Json::Value& bundle_meta, const std::vector<std
     }
   }
 }
+
+std::tuple<OfflineUpdateAppsShortlistType, std::set<std::string>> getAppsFetchedForOfflineUpdate(
```

Comment:
Since there is some non-trivial logic in here, adding one LOG entry (at least LOG_DEBUG) on each return statement, in line with the text in the comments, might help users to identify issues with their bundles.

Reply:
I agree, will add logs to it, maybe even `info` logs and/or `warn`.

This is still WIP, I am going to add unit tests.

Reply:
@detsch I added logs and unit tests.